voice-agent-tester/kubernetes/README.md

```diff
@@ -0,0 +1,24 @@
+# Deploy Microservice on Kubernetes
+
+This is a simple setup for deploying the Agent Tester microservice to a kubernetes cluster on IBM Cloud. See the `concurrent_scaling` directory which contains deployment scripts for scaling and running multiple concurrent calls at the same time. The other scripts are just for deploying them all on one worker.
+
+## Prerequisites
+
+You must have an account for Watson Services Text to Speech and Speech to Text and also a kubernetes cluster service. Configure the following CLI tools so that you can access your Kubernetes cluster on IBM Cloud through the command line
```

Comment:
When you mention services you should use names and correct links so people can look up more information on them.

"You must have an account for Watson Services Text to Speech and Speech to Text and also a kubernetes cluster service"

=> You must have an account for [Watson Speech-To-Text](https://cloud.ibm.com/catalog/services/speech-to-text) and [Watson Text-To-Speech](https://cloud.ibm.com/catalog/services/text-to-speech) and also an [IBM Cloud Kubernetes cluster](https://cloud.ibm.com/kubernetes/catalog/cluster/create).